.github/workflows/check-infra-deploy-status.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow checks the status of infrastructure deployments to see whether
+# infrastructure code configuration matches the actual state of the infrastructure.
+# It does this by checking that Terraform plans show an empty diff (no changes)
+# across all root modules and backend configurations.
+name: Check infra deploy status
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every day at 00:00 UTC
+    - cron: "0 0 * * *"
```

Comment:
Nit: Suggest 7am UTC which is between 12am PT and 3am ET. 12am UTC is still within business hours PT (5pm).